docs/tutorials/delete-a-user-levibeverly.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: page
+---
+
+# Tutorial: Delete a user
+
```

Comment:
Somewhere you might describe how long this tutorial will take. If both options take about the same time, you could add that time in the introduction. If one is radically different from the other, add the time to each option.